settings.py

```diff
@@ -1,6 +1,7 @@
 from flask import Flask
 from flask_cors import CORS
 from flask_login import LoginManager
+from flask_mail import Mail
```

Comment:
To samo, nie ma flask_mail w paczce - upewnij się, że jest wszystko w environment.yml